components/atom/label/src/index.tsx

```diff
@@ -0,0 +1,57 @@
+import type {FC, LabelHTMLAttributes} from 'react'
+
+import cx from 'classnames'
+
+import {CLASSNAME, FONT_SIZES, TYPES} from './settings'
+
+type FontSize = typeof FONT_SIZES[keyof typeof FONT_SIZES]
+type Type = typeof TYPES[keyof typeof TYPES]
+
+export interface AtomLabelProps extends Pick<LabelHTMLAttributes<HTMLLabelElement>, 'htmlFor' | 'onClick'> {
+  /**
+   * used as for attribute. Must be the same as the form element id
+   */
+  name: string
+  /**
+   * The label itself
+   */
+  text: string
+  /**
+   * Allows label to be displayed inline to de left
```

Comment:
Typo `Allows label to be displayed inline to de left` -> `Allows label to be displayed inline to the left`

Reply:
Copypasted, sorry 😂